packages/odyssey-design-tokens/src/components/button.json

```diff
@@ -1,60 +0,0 @@
-{
```

Comment:
Does the current setup allow for us to house component tokens in style dictionary?

I don't think it's necessary here, but it could be helpful when we want to canonize other theme variables.

Reply:
no and this is intentional, [I mentioned this tradeoff in the MVP writeup too](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2401674612/Odyssey+Theming+MVP). The component tokens are defined within the theme reducer alongside the component code and styles under this design. Consider that when downstream teams build their own themeable components it won't make sense to have them live upstream here either.

Reply:
@edburyenegren-okta I've been thinking about this in more detail, lets talk more at standup about tradeoffs and benefits here